packages/components/scrollbar/src/scrollbar.vue

```diff
@@ -6,6 +6,7 @@
         ref="resizeRef"
         :class="resizeKls"
         :style="viewStyle"
+        tabindex="0"
```

Comment:
@tolking like that?

Reply:
IMO, moving `tabindex` to `wrapRef` may be more appropriate,  because we need to improve the `:focus-visible` style.

It should be better as an external attribute, and developers can cover it.